rs/cli/src/main.rs

```diff
@@ -327,6 +330,32 @@ async fn async_main() -> Result<(), anyhow::Error> {
                     ..Default::default()
                 }, cli_opts.simulate).await
             }
+            cli::Commands::Proposals(p) => match &p.subcommand {
+                cli::proposals::Commands::Pending => {
+                    let nns_url = target_network.get_nns_urls().first().expect("Should have at least one NNS URL");
+                    let client = GovernanceCanisterWrapper::from(CanisterClient::from_anonymous(nns_url)?);
+                    let proposals = client.get_pending_proposals().await?;
+                    let proposals = serde_json::to_string(&proposals).map_err(|e| anyhow::anyhow!("Couldn't serialize to string: {:?}", e))?;
```

Comment:
```suggestion
                    let proposals = serde_json::to_string_pretty(&proposals).map_err(|e| anyhow::anyhow!("Couldn't serialize to string: {:?}", e))?;
```

Reply:
Wouldn't it be better to pipe it to `jq` if you need pretty string? The intention for this is just to dump the proposals (similar as to `dump-registry`) and pipe consume it with something else. I doubt it that this will be used from an end user being an engineer. WDYT?

Reply:
A nice output is preferable to a noisy blob.  We _can_ pipe things to other things, but I would prefer not to _have_ to do that.

Reply:
It will be a noisy blob anyway, JSON serializers serialize array's as each element in a new line. Since the payload is an array of bytes you get a really large and ugly looking array (which means nothing) and it takes a lot of the terminal. I think its better to have a default and consume it if you really need it to. Which I doubt anyone will without further filtering and selecting relevant data (also with jq).

Reply:
The question is really who is the expected user of this output?
A machine doesn't care if it's pretty or not, and we don't care (that much) about the bandwidth either. If the bandwidth is an issue, let's use json+gzip. But I don't think it is something we need to do.
A human would care if the output is pretty or not, and would have to use jq every time, if they would ever use the output. Adding this requirement of the jq doesn't make much sense IMHO.
Personally, I do expect the output to be viewed by user(s), at least sometimes.

Reply:
I would prefer to create a different command for the users since this is just a map from gov canister. This was created with the vision of replacing Airflow's dependency to public dashboard. I really doubt that user needs all of this data that is here.

Reply:
I'm okay if we add a different user-facing command. Do you already have plans on how that will look like?

Reply:
Yes, as of now it will include:
1. greater limit (you could request more than 100 proposals)
2. would work on `including` rather than `excluding`
3. would have way less data mapped to more friendly interface since now we just return what we map from gov canister. (Things like binary payload and so on)